src/scripts/main.js

```diff
@@ -3,6 +3,20 @@
 const people = require('./lib/people');
 
 // eslint-disable-next-line no-console
-console.log(people); // you can remove it
+// console.log(people); // you can remove it
 
-// write your code here
+const str = document.querySelector('.dashboard');
```

Comment:
```suggestion
const table = document.querySelector('.dashboard');
```
Чому str? Давай використовувати більш зрозумілі і логічні назви